diff --color -bur dxvk_old/src/util/config/config.h dxvk/src/util/config/config.h
--- dxvk_old/src/util/config/config.h	2024-07-04 22:19:54.117340772 -0400
+++ dxvk/src/util/config/config.h	2024-07-04 22:20:44.110686537 -0400
@@ -1,5 +1,6 @@
 #pragma once

+#include <cstdint>
 #include <string>
 #include <unordered_map>

diff --color -bur dxvk_old/src/util/util_bit.h dxvk/src/util/util_bit.h
--- dxvk_old/src/util/util_bit.h	2024-07-04 22:19:54.124007497 -0400
+++ dxvk/src/util/util_bit.h	2024-07-04 22:21:22.093936014 -0400
@@ -16,6 +16,7 @@
 #include "util_likely.h"
 #include "util_math.h"

+#include <cstdint>
 #include <cstring>
 #include <type_traits>
